### DISCUSSION TITLE
Backup of tables

### DISCUSSION THREAD

**Discussion by akshayparseja** [Altinity/clickhouse-backup#1057] [Q&A] [answered]
if i want to do a migration of tables from a clickhouse to another without migrating its mvs , we would like only tables since mvs could conflict clickhouse consumer .
can we do so?
I have alot of dbs with tables which i want to migrate , a lot have associated mv as a kafka consumer which we want to skip

**Comment by Slach** [comment, marked as answer]
use 
```
clickhouse:
  skip_table_engines:
  - Kafka
```

**Reply by akshayparseja** [reply]
Wont that skip backup of kafka engines?, I dont want to touch the CH which is doing the backup.
I want to restore a backup of a CH which has tables and MVs in it , but as part of restoration i want to skip the MVs since there will be conflict between Consumer. Both will consume using same name on same topic.

We plan on creating MVs manually , just mgirate tables of dbs

**Reply by Slach** [reply]
`skip_table_engines` allow you avoid create engine=Kafka tables during restore

MV this is just a trigger which pull engine=Kafka only if it present in system
